app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    """this IF-block just will rename file without moving somewhere"""
+    parts = command.split()
+
+    if len(parts) == 3 and parts[0] == "mv":
```

Comment:
You keep that indent across wholefunction, better do the opposite and return fail

Reply:
.